src/update-feeds-loops/update-feeds-loops.ts

```diff
@@ -183,7 +183,7 @@ export const runUpdateFeeds = async (providerName: string, chain: Chain, chainId
           provider,
           chainId,
           firstBatchBlockNumber
-        ).catch((error) => error);
+        ).catch((error: any) => error);
```

Comment:
In new version 0.6.1 of `@total-typescript/ts-reset`, catch resolves output as unknown rather than any.